lib/Types/Attean.pm

```diff
@@ -56,6 +64,46 @@ L<XML::Namespace> and strings.
 
 Additionally, a C<ScalarRef> can be coerced into a C<data> URI.
 
+=item C<< AtteanBlank >>
+
+A role type for L<Attean::API::Blank>.
+
+=item C<< AtteanLiteral >>
+
+A role type for L<Attean::API::Literal>.
+
+=item C<< AtteanSubject >>
```

Comment:
Other possible names:

- `AtteanTermSubject`
- `AtteanSubjectTerm`

Same for the

- `AtteanPredicate`
- `AtteanObject`
- `AtteanGraph`

below.

Reply:
I'm fine with this as-is. Will merge. Thanks!